libDCM/hilsim.c

```diff
@@ -21,10 +21,121 @@
 

```

Comment:
Have considered this quite extensively, and believe it almost certainly does not belong in libUDB either. Please note the code this module contains was segregated out of libDCM/gpsParseUBX.c, and so if we could consider it an incremental step to improving overall modularity, I'd like to leave it as is for now.